server/routes/notification.js

```diff
@@ -0,0 +1,19 @@
+const { Route53Resolver } = require("aws-sdk");
+const express = require("express");
+const router = express.Router();
+const {
+  getAllNotifications,
+  getAllUnreadNotifications,
+  createNewNotification,
+  markNotificationAsRead,
+} = require("../controllers/notifications");
+// works
```

Comment:
These comments aren't necessary, code reviewers will always assume you have tested all aspects of your feature and they are working. If you haven't but want review, you should put up a draft PR. All that to say, it's fine to remove these and don't worry about mentioning you've tested in the PR (because it's assumed)

One time I do still mention my testing personally, is if I'm pushing a feature that will have broad effects I list all of the tests I have done and confirmed, that way a reviewer can easily know my test coverage and if I missed any cases.